cli/features/tutorial.feature

```diff
@@ -175,11 +176,10 @@ Feature: Following the tutorial
               - '27017:27017'
             online-text: 'waiting for connections'
       """
-    When running "exo setup" in this application's directory
     And running "exo test" in this application's directory
-    Then it prints "todo-service works" in the terminal
+    # Then it prints "todo-service works" in the terminal
```

Comment:
I think I need help with fixing this test. So the new `exo-run` also launches the app and I think `exo-test` also does that. So I got this error if I keep the original steps:
```
exo-test  Error: Error: (HTTP code 500) server error - driver failed programming external connectivity on endpoint todo-service-test-mongo (a7bae7fbc65152f46db7f28968da29f37335add98a7586cc95313e58b1ccb6c4): Bind for 0.0.0.0:27017 failed: port is already allocated 
```
I tried not doing `exo-run`, and got this:
```
exo-test  Testing application 'todo-app'
    exo-test  html-server has no tests, skipping
    exo-run Attaching to exocom0.22.1
     exocom Exocom 0.22.1
     exocom 
     exocom 'exocom' is running
    exo-run all dependencies online
     exocom ExoCom WebSocket listener online at port 80
     exocom ExoCom HTTP service online at port 80
    exo-run exocom0.22.1 is up-to-date
    exo-run Creating html-server
todo-service  bash: node_modules/cucumber/bin/cucumber.js: No such file or directory
todo-service  
    exo-test  todo-service is broken
    exo-run Attaching to html-server
    exo-run html-server exited with code 137
    exo-run 
    exo-run exocom0.22.1 exited with code 137
    exo-test  Tests failed
```
I might be wrong but I think this is because `exo-test` is run locally instead of inside the docker image, and we don't run startup or dockerfile commands locally.